nemo/collections/asr/metrics/wer_bpe.py

```diff
@@ -12,18 +12,87 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+from dataclasses import dataclass
 from typing import List
 
 import editdistance
 import torch
 from torchmetrics import Metric
 
-from nemo.collections.asr.metrics.wer import move_dimension_to_the_front
-from nemo.collections.asr.parts.utils.rnnt_utils import Hypothesis
+from nemo.collections.asr.metrics.wer import AbstractCTCDecoding, CTCCharDecodingConfig
 from nemo.collections.common.tokenizers.tokenizer_spec import TokenizerSpec
 from nemo.utils import logging
 
 
+class CTCBPEDecoding(AbstractCTCDecoding):
+    """
```

Comment:
Similar docs as AbstractCTCDecoding?

Reply:
Yes, docs should be same. Normally I would not copy paste but use python to stitch together the docs but sphinx / RTD doesn't like that.